junit5-api/src/main/java/org/junit/gen5/api/extension/ParameterResolver.java

```diff
@@ -32,61 +31,46 @@
  * <p>Implementations must provide a no-args constructor.
  *
  * @since 5.0
- * @see #supports(Parameter, Optional, ExtensionContext)
- * @see #resolve(Parameter, Optional, ExtensionContext)
+ * @see #supports(ParameterContext, ExtensionContext)
+ * @see #resolve(ParameterContext, ExtensionContext)
+ * @see ParameterContext
  */
 @API(Experimental)
 public interface ParameterResolver extends Extension {
 
 	/**
-	 * Determine if this resolver supports resolution of the given {@link Parameter}
-	 * for the supplied {@code target} and {@link ExtensionContext}.
+	 * Determine if this resolver supports resolution of the given
+	 * {@link ParameterContext} in the supplied {@link ExtensionContext}.
 	 *
-	 * <p>The {@link java.lang.reflect.Method} or {@link java.lang.reflect.Constructor}
-	 * in which the {@code parameter} is declared can be retrieved via
-	 * {@link Parameter#getDeclaringExecutable()}.
```

Comment:
I think this tip is very useful for the average extension developer.

As such, I would not remove it but rather reword it to point to the new `getDeclaringExecutable()` method you've introduced in the `ParameterContext`.

The same applies to the `resolve()` method.